frontend/angular/README.md

```diff
@@ -0,0 +1,23 @@
+## Introduction
+Angular is a popular front-end framework made by Google. Like other popular front-end frameworks, it uses a component-based architecture to structure apps.
+
+If you are a beginner for Angular and wonder how to start, This article is for you and we focus only on basic Angular App with one page. Once finished our app looks like below where we can add items, edit, delete items.
+
+## How to get started with Angular
+
+-   Install [Nodejs](https://nodejs.org/en/download/)
+-   Install Angular CLI
+-   Install Typescript globally
```

Comment:
Don't do duplication. You can combine this with the command to install it.

Reply:
Done sir, please review the same @harsh-2711